src/koheesio/spark/readers/databricks/autoloader.py

```diff
@@ -96,6 +105,22 @@ def validate_format(cls, format_specified):
                 format_specified = getattr(AutoLoaderFormat, format_specified.upper())
         return str(format_specified.value)
 
+    @field_validator("schema")
+    def validate_schema(cls, schema_specified):
+        """Validate `schema` value"""
+        schema = schema_specified
+        if isinstance(schema, StructType):
+            return schema
+
+        if schema is not None:
+            schema_path = Path(schema)
+            if schema_path.exists():
+                schema = schema_path.read_text()
+            else:
+                raise FileNotFoundError(f"Schema file not found at path {schema}")
+            schema = StructType.fromJson(json.loads(schema))
```

Comment:
I think it might be benefitial to wrap this into `try...except`

Reply:
Do you prefer the implementation in [rest_api](https://github.com/Nike-Inc/koheesio/blob/main/src/koheesio/spark/readers/rest_api.py#L109) to keep things in line ?

Reply:
Sure. Lets keep it consistent.